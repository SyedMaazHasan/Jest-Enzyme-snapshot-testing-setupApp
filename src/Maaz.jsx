import React, { Component } from "react";

class Maaz extends Component {
  render() {
    return <div>my name is maaz</div>;
  }
}

export default Maaz;
