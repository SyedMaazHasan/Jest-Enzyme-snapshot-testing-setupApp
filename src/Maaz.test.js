import renderer from "react-test-renderer";
import Maaz from "./Maaz";
import React from "react";

it("snapshot testing", () => {
  const tree = renderer.create(<Maaz />).toJSON();
  expect(tree).toMatchSnapshot();
});
