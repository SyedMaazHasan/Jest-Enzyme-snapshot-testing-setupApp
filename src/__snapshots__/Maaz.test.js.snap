// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing 1`] = `
<div>
  my name is maaz
</div>
`;
